Microsoft.NET.Build.Containers/ContainerHelpers.cs

```diff
@@ -136,11 +136,28 @@ public static bool NormalizeImageName(string containerImageName, [NotNullWhen(fa
             if (Char.IsUpper(containerImageName, 0))
             {
                 containerImageName = Char.ToLowerInvariant(containerImageName[0]) + containerImageName[1..];
-            } else if (!Char.IsLetterOrDigit(containerImageName, 0)) {
+            }
+            else if (!Char.IsLetterOrDigit(containerImageName, 0))
+            {
                 throw new ArgumentException("The first character of the image name must be a lowercase letter or a digit.");
             }
             normalizedImageName = imageNameCharacters.Replace(containerImageName, "-");
             return false;
         }
     }
+
+    public static bool TryParsePort(string input, [NotNullWhen(true)] out Port? port)
```

Comment:
Can we get a test for this?

Reply:
can do!

Reply:
Excellent reminder to test because I'd inverted the boolean condition at the end of the method.